jsonschema2pojo-gradle-plugin/example/android/lib/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,4 @@
-<manifest package="jsonschema2pojo.joelittlejohn.github.com.androidlibexample">
```

Comment:
Does the namespace declaration make this redundant?  Why is this package declaration leaving?

Reply:
That is the new mandatory rules for android.
The recent android compilers block the build if you don't respect them

Reply:
Thanks for the insight.